weave-js/src/components/PagePanelComponents/Home/Browse3/pages/CallPage/ObjectViewerSection.tsx

```diff
@@ -187,21 +203,24 @@ const ObjectViewerSectionNonEmpty = ({
   return (
     <Box sx={{height: '100%', display: 'flex', flexDirection: 'column'}}>
       <TitleRow>
-        <Title>{title}</Title>
-        <Button
-          variant="quiet"
-          icon="row-height-small"
-          active={mode === 'collapsed'}
-          onClick={onClickCollapsed}
-          tooltip="View collapsed"
-        />
+        <Title
+          onClick={() => setMode(mode === 'hidden' ? 'collapsed' : 'hidden')}>
```

Comment:
total nit but i've been thinking that 1 liner functions are more aesthetic than I previously thought, we might consider adding a local `isModeHidden` fn like:

```javascript
function isModeHidden(mode) {
   return mode === 'hidden'
}
function isModeExpanded...
```

Reply:
and then obviously use it everywhere we test the mode state